### DISCUSSION TITLE
Issues with Mastodon on Debian 12

### DISCUSSION THREAD

**Discussion by compi2k** [mastodon/mastodon#29080] [Q&A] [answered]
I installed Mastodon on Debian 12 following the official guide. I try to avoid Debian 11, since it will be EOL rather soon. The installation worked without a hitch, but this is what I get:
![image](https://github.com/mastodon/mastodon/assets/65253326/d9a132ea-c8c4-429e-91e9-143e1be47f1c)

I searched the web and found this article which helped to get the whole thing to show properly: https://github.com/mastodon/mastodon/discussions/23036#discussioncomment-4687313

But after a reboot I was hit with the error screen where the elephant hits the keyboard. 
Nothing helps, except this: https://github.com/mastodon/mastodon/discussions/23036#discussioncomment-7158251
When I run `RAILS_SERVE_STATIC_FILES=true RAILS_ENV=production bundle exec rails server ` in the server works fine, but adding `RAILS_SERVE_STATIC_FILES=true` to `.env.production` does not work.

What can I do? Is mastodon really that bad on current OS versions?

**Comment by snullp** [comment, marked as answer]
This is because nginx unable to read /home/mastodon directory. chmod +rx /home/mastodon/ works for me.

**Comment by compi2k** [comment]
That seemed to work - kinda.
It broke immediately and told me to wait for verification.
![image](https://github.com/mastodon/mastodon/assets/65253326/a7bfb505-b624-407e-9103-71a25bf3f15d)

**Reply by snullp** [reply]
I believe this is a separate issue - my comment can only address the blank page issue.

**Reply by compi2k** [reply]
yes, in the meantime I worked it out. I had to manually approve my admin account with tootctl.
